LICENSE.md

```diff
@@ -100,17 +100,17 @@ License: [The Apache Software License, Version 2.0](http://www.apache.org/licens
 
 Mapbox Maps Android uses portions of the Mapbox Android Commmon OkHttp SDK.
 
-URL: [https://github.com/mapbox/mapbox-sdk](https://github.com/mapbox/mapbox-sdk)
+URL: [https://github.com/mapbox/mapbox-sdk-common](https://github.com/mapbox/mapbox-sdk-common)
 
-License: [BSD](https://opensource.org/licenses/BSD-2-Clause)
+License: [Mapbox Terms of Service](https://www.mapbox.com/legal/tos)
 
 ===========================================================================
 
 Mapbox Maps Android uses portions of the Mapbox Android Commmon SDK.
 
-URL: [https://github.com/mapbox/mapbox-sdk](https://github.com/mapbox/mapbox-sdk)
+URL: [https://github.com/mapbox/mapbox-sdk-common](https://github.com/mapbox/mapbox-sdk-common)
 
-License: [BSD](https://opensource.org/licenses/BSD-2-Clause)
+License: [Mapbox Terms of Service](https://www.mapbox.com/legal/tos)
```

Comment:
Hey folks, just seeing that you have changed the license from BSD to the Mapbox Terms of Services.

It would be great if you could call this out in your release notes or changelog.

What are the implications here for
- code previously contributed under the BSD license
- code in the future contributed under the new license

because the linked terms of service are not talking about code requirements or restrictions.

If someone contributes to this code base tomorrow, under what license do the contribution falls? What are the requirements, the restrictions, and so on?

Reply:
Hi @daniel-j-h. I'm not sure what implications you're referring to for older BSD code. Can you elaborate?

The lines attached to this thread describe a dependency change; the new dependency is in a private repo, so we don't expect public contributions to it. If you're asking about future contributions to this repo, please see https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement. The change to this repo's license was called out in the changelog at the time it was made.